snapshot/snapshots/fake_checkouter.go

```diff
@@ -15,6 +15,12 @@ type noopCheckouter struct {
 	path string
 }
 
+// have to have a valid path in the checkouter otherwise exec'ing a command
+// from simple.go doesn't work
+func MakeNoOpCheckouter() snapshot.Checkouter {
```

Comment:
I think TempCheckouter can be used for this case (and we probably don't want a Checkouter pointing to cwd).